src/main/java/org/opentripplanner/model/plan/Leg.java

```diff
@@ -171,6 +171,8 @@
 
    public Boolean rentedBike;
 
+   public Set<String> bikeRentalNetworks = new HashSet<>();
```

Comment:
A minor comment about DTOs and Collections:

Leg is a DataTransferObject(DTO), not use for much bussiness logic. Hence, in OTP we mostly use List even for collection with unique values. This allow navigation in the collection. This is not a big deal, but if a Set is used in a DTO it should trigger the question: "What is this used for", after all restrictons on the values are allpiled to the collection BEFORE creating the DTO. The DTO should be as general as possible. I am happy to ues both Set and/or List but it should be used consistent - and maybe also with a comment why - if set is used.